share/instruments/instruments.xml

```diff
@@ -10179,8 +10153,6 @@
                   <genre>popular</genre>
             </Instrument>
             <Instrument id="electric-guitar">
-                  <!-- don't remove this instrument, it is used in ...mtest/libmscore/links/tst_links.cpp
-                 and ...mtest/libmscore/midi/tst_midi.cpp for testing purposes -->
```

Comment:
I'd rather have this comment kept

Reply:
It has now been retained a couple of lines down.